tests/test_core.py

```diff
@@ -199,6 +199,26 @@ def test_generate_from_jams(atol=1e-5, rtol=1e-8):
                          pitch_shift=('uniform', -1, 1),
                          time_stretch=('uniform', 0.8, 1.2))
 
+
+        def _validate_soundscape_and_event_audio(orig_wav_file, gen_wav_file):
+            # validate audio
+            orig_wav, sr = soundfile.read(orig_wav_file.name)
+            gen_wav, sr = soundfile.read(gen_wav_file.name)
+            assert np.allclose(gen_wav, orig_wav, atol=atol, rtol=rtol)
+
+            # validate event audio is trimmed and sums to trimmed soundscape
```

Comment:
this comment discusses trimming but this code is actually trimming agnostic, we should probably remove reference to trimming from here

Reply:
Good point, will update the comment.